baseplate/clients/thrift.py

```diff
@@ -2,6 +2,7 @@
 import inspect
 import sys
 
+from math import ceil
```

Comment:
I think this import should be on the first group?

Reply:
It's good where it is. We're configured to separate `import` and `from ... import` into two blocks for each of the three groups of imports.